src/main/java/frc/robot/common/Odometry.java

```diff
@@ -94,10 +95,13 @@ public Pose2d getPose() {
         return poseEstimator.getEstimatedPosition();
     }
 
+    // It's helpful to name boolean functions starting with 'is' instead of 'get'
+    // isRotCorrected reads clearer
     public boolean getRotCorrected() {
         return rotCorrected;
     }
 
+    // Why are there two functions that return seemingly the same values?
```

Comment:
These values are similar but not quite the same, rotCorrected is turned false every update, so it only is true the first time the rot is corrected. The drive subsystem uses this to know to reset the lockedRot. correctRot is held true or false for as long is the rot is accurate or not. I will change the names to be something more clear though